models/public/yolo-v4-tf/accuracy-check.yml

```
@@ -0,0 +1,97 @@
+models:
+  - name: yolo_v4_tf
+    launchers:
+      - framework: tf
+        model:   yolo-v4.pb
+        adapter:
+          type: yolo_v3
+          anchors: 12,16,19,36,40,28,36,75,76,55,72,146,142,110,192,243,459,401
+          num: 9
+          coords: 4
+          classes: 80
+          threshold: 0.001
+          anchor_masks: [[0, 1, 2], [3, 4, 5], [6, 7, 8]]
+          raw_output: True
+          output_format: HWB
+          do_reshape: True
+          cells: [76, 38, 19]
+          outputs:
+            - conv2d_93/BiasAdd
+            - conv2d_101/BiasAdd
+            - conv2d_109/BiasAdd
+        inputs:
+          - name: 'image_input'
+            type: INPUT
+            shape: [1, 608, 608, 3]
+        outputs:
+          - conv2d_93/BiasAdd
+          - conv2d_101/BiasAdd
+          - conv2d_109/BiasAdd
+    datasets:
+      - name: ms_coco_detection_80_class_without_background
+        preprocessing:
+          - type: bgr_to_rgb
+          - type: normalization
+            std: 255.0
+          - type: resize
+            size: 608
+        postprocessing:
+          - type: resize_prediction_boxes
+          - type: filter
+            apply_to: prediction
+            min_confidence: 0.001
+            remove_filtered: true
+          - type: nms
+            overlap: 0.5
+          - type: clip_boxes
+            apply_to: prediction
+        metrics:
+          - type: map
+            integral: 11point
+            ignore_difficult: true
+            presenter: print_scalar
+          - type: coco_precision
+            max_detections: 100
+            threshold: 0.5
+  - name: yolo_v4
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: yolo_v3
+          anchors: 12,16,19,36,40,28,36,75,76,55,72,146,142,110,192,243,459,401
+          num: 3
+          coords: 4
+          classes: 80
+          threshold: 0.001
+          anchor_masks: [[0, 1, 2], [3, 4, 5], [6, 7, 8]]
+          raw_output: True
+          outputs:
+            - conv2d_93/BiasAdd/Add
+            - conv2d_101/BiasAdd/Add
+            - conv2d_109/BiasAdd/Add
+    datasets:
+      - name: ms_coco_detection_80_class_without_background
+        preprocessing:
+          - type: resize
+            size: 608
+        postprocessing:
+          - type: resize_prediction_boxes
+          - type: filter
+            apply_to: prediction
+            min_confidence: 0.001
+            remove_filtered: true
+          - type: nms
+            overlap: 0.5
+          - type: clip_boxes
+            apply_to: prediction
+        metrics:
+          - type: map
+            integral: 11point
+            ignore_difficult: true
+            presenter: print_scalar
+          - type: coco_precision
+            max_detections: 100
+            threshold: 0.5
+          - type: coco_precision
+            max_detections: 100
+            threshold: '0.5:0.05:0.95'
```

It is hard to match metric results after evaluation (you have 2 metrics with the same name, but different values)
```suggestion
          - name: AP@0.5 
            type: coco_precision
            max_detections: 100
            threshold: 0.5
          - name: AP@0.5:0.05:0.95 
            type: coco_precision
            max_detections: 100
            threshold: '0.5:0.05:0.95'
```